### ISSUE TITLE
The profile specified in the "Chrome profile" is ignored and a new profile is created

### ISSUE THREAD

**Issue by batgh** [g3rv4/OnChrome#8]
Hi, this extension is exactly what I was looking for and I enjoy this tool but one issue I can't find the solution.
If "Chrome profile" is blank, it works fine however if I use a specific profile I want to use, the extension always opens "User _<N>_" profile. I use multiple profiles in Chrome and try to use one by one but none of them works. And once Chrome is restarted, new "User _<N + 1>_" profile is created.

OS: Mac BigSur (11.5.2)
Chrome: 93.0.4577.63 (always updated)
Firefox: 91.0.2
The native app is installed via OnChrome.pkg, not brew.

Let me know if you need further information.

**Comment by g3rv4** [comment]
sorry for the delay... I can't repro. Unfortunately, figuring out the Profile directory isn't as straighforward as it should be, but this trick should help: https://www.howtogeek.com/255653/how-to-find-your-chrome-profile-folder-on-windows-mac-and-linux/

TL;DR:

1. Open Chrome with the profile you want to use
2. Visit `chrome://version`
3. Check the value of `Profile Path`. You will see something like `/Users/<yourUser>/Library/Application Support/Google/Chrome/Profile 1`. In this case, you need to enter `Profile 1` on the extension

Let me know if that works, I promise I'll take less than a year to reply this time.

**Comment by batgh** [comment]
Thanks for the information. Uisng "Profilel <N>" works fine! (I have 10
profiles).
I thought the name is supposed to be a name listed in the profiles...


2022年7月11日(月) 10:08 Gervasio Marchand ***@***.***>:
